BUILD.bazel

```diff
@@ -424,52 +424,52 @@ ray_cc_library(
     name = "plasma_client",
     srcs = [
         "src/ray/object_manager/plasma/client.cc",
-        "src/ray/object_manager/plasma/connection.cc",
```

Comment:
Thanks! It's amazing --- as of now we're just spending time double building the files.